src/miniscript/types/extra_props.rs

```diff
@@ -93,9 +93,8 @@ impl TimeLockInfo {
     }
 }
 
-/// Structure representing the extra type properties of a fragment which are
-/// relevant to legacy(pre-segwit) safety and fee estimation. If a fragment is
-/// used in pre-segwit transactions it will only be malleable but still is
+/// Structure representing the extra type properties of a fragment. If a fragment
+/// is used in pre-segwit transactions it will only be malleable but still is
```

Comment:
I think we should delete the second sentence. I can't see how it's related to this structure.